### ISSUE TITLE
Sprite3D Attribute not found: a_normal

### ISSUE THREAD

**Issue by milos1290** [cocos2d/cocos2d-x#12907]
Exported 3d model from Blender produces this warning if there are no lights in the scene

cocos2d: warning: Attribute not found: a_normal

It happens because model contains normals, but if there are no light sources in the scene, shader that doesn't have normals will be used and this warning will pop up, when trying to bind attributes.

**Comment by fusijie** [comment]
yep...
But it's ok, can't find normal attribute is ok... and normal is not used if the scene has no light source..
If you wanna to use normal attribute for other purpose, you should define your own shaders.
Or dose it cause any bugs?

**Comment by ginobean** [comment]
Would it make sense to change this warning from CCLOG to CCLOGINFO level or even take it out altogether ? I have a case where I'm literally creating dozens of these objects, so I'm getting dozens of these warning messages flooding my console output. 

What would happen if you disabled/removed this warning message altogether ?

**Comment by milos1290** [comment]
Yep exactly the same problem, my console is getting spammed with this log message. It would be great if we can somehow disable this.
